packages/react-server/core/util/PageUtil.js

```diff
@@ -61,7 +62,6 @@ var PAGE_METHODS = {
 	getBase            : [() => null, Q],
 	getBodyClasses     : [() => [], Q],
 	getElements        : [() => [], standardizeElements],
-	getAboveTheFoldCount : [() => 1, _ => _],
```

Comment:
Glad I never updated the tabling around this.  It's almost like I knew it was an ugly hack that should be replaced... 👹